pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
     gnupg                     # undefined dependencies
     xapian gmime talloc zlib  # dependencies described in INSTALL
     perl
-    pythonPackages.python
+    (pythonPackages.python.withPackages (ps: [ps.setuptools ps.cffi]))
```

Comment:
Is python used at runtime here or at compile time?

Reply:
I’ll double check. I’ve been using a 0.33 RC, and I noticed an error somewhere that was resolved by this change but I don’t recall where it was (eg on the `notmuch` package, or the associated Python module?). I might not get keyboard time until Wednesday, so if anyone has a chance to check, we can fix this if it needs fixing.

Reply:
I've changed it to add `pythonPackages.cffi` to `nativeBuildInputs` which seems to address the error I saw before.